.eslintrc.js

```diff
@@ -1,17 +1,46 @@
 /** @type {import('eslint').Linter.Config} */
+const year = new Date().getFullYear();
 module.exports = {
     root: true,
-    extends: [
-        './configs/base.eslintrc.json',
-        './configs/warnings.eslintrc.json',
-        './configs/errors.eslintrc.json'
-    ],
-    ignorePatterns: [
-        '**/{node_modules,lib}',
-        'plugins'
-    ],
+    extends: ['./configs/base.eslintrc.json', './configs/warnings.eslintrc.json', './configs/errors.eslintrc.json'],
+    ignorePatterns: ['**/{node_modules,lib}'],
     parserOptions: {
         tsconfigRootDir: __dirname,
-        project: 'tsconfig.json'
-    }
+        project: 'tsconfig.eslint.json'
+    },
+    overrides: [
+        {
+            files: ['*.spec.ts'],
+            rules: {
+                'no-unused-expressions': 0,
+                'no-invalid-this': 0
+            }
+        },
+        {
+            files: ['*.ts', '*.tsx'],
+            rules: {
+                // eslint-plugin-header
+                'header/header': [
+                    2,
+                    'block',
+                    [
+                        {
+                            pattern: '[\n\r]+ \\* Copyright \\([cC]\\) \\d{4}(-\\d{4})? .*[\n\r]+',
+                            template: `********************************************************************************
+  * Copyright (c) ${year} EclipseSource and others.
+  *
+  *
```

Comment:
One new line is sufficient here

```suggestion
  *
```